### PR TITLE
feat(deployment-stage-item): add integration

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-655)

- Add component BadgeService
- Add item design for the deployment stage

Capture d’écran 2023-02-23 à 18 43 06
Capture d’écran 2023-02-23 à 18 46 41

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(deployment-stage-item): add integration - 2023-02-23T17:43:39Z](https://new.console.qovery.com/organization/141c07c8-0dd9-4623-983b-3fdd61867255/project/977bd069-20f7-4b24-bd95-28b44174cf43/environment/cbd1a6ca-d63b-40ec-8e99-5f3cbd6c70d6/services/general)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://zcbd1a6ca-z3215061a-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://zcbd1a6ca-z4caeb4c8-gtw.za8ad0659.bool.sh)

File: libs/pages/services/src/lib/ui/page-settings-deployment-pipeline/draggable-item/draggable-item.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const serviceType = getServiceType(service)
+
+  const classNameItem = (isDragging: boolean) =>
+    `flex items-center drop-shadow-item-deployment-group bg-element-light-lighter-100 rounded px-2 py-3 border ${
+      isDragging ? 'border-2 border-success-500' : 'border-element-light-lighter-400'
+    }`
+
+  const contentWithParams = serviceType === ServiceTypeEnum.DATABASE
+
+  const content = (name: string, type: string, mode: string) => {
```
Comment (bdebon) [inline libs/pages/services/src/lib/ui/page-settings-deployment-pipeline/draggable-item/draggable-item.tsx:36]: Why don't you put the content directly in the return part of the component ? Why do you need to extract this part in a function?

**Review by bdebon — APPROVED** [review]
Super cool @RemiBonnet ! It's so nice to have the badge service component. It is useful for the whole project and work so well in these stage columns.
I like it.
Just made a small commit to enrich the widget in the storybook, and asked a small question about a part I was not really sure to understand. Gg

File: libs/pages/services/src/lib/ui/page-settings-deployment-pipeline/draggable-item/draggable-item.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/pages/services/src/lib/ui/page-settings-deployment-pipeline/draggable-item/draggable-item.tsx:36]: I took it out to make it easier to read, but I can put it inside if you want

**Pull request merged**